docs/css/main.css

```diff
@@ -196,11 +196,14 @@ a.anchorjs-link:hover { text-decoration: none; }
   width: 210px;
   float: left;
   position: fixed;
+  overflow-y: scroll;
+  top: 0;
+  bottom: 0;
 }
 
 .left-menu {
   padding: 0px;
-  width: 199px;
+  width: 399px;
```

Comment:
Sorry for leaving a comment in an old PR but @dilipbiswal do you remember why you had to do this? It 
![Screen Shot 2020-11-12 at 1 16 25 PM](https://user-images.githubusercontent.com/6477701/98894991-5eb78f00-24e9-11eb-9c1d-bc6003591a1b.png)

This makes each title look sort of truncated. I was thinking it should be like:

![Screen Shot 2020-11-12 at 1 16 46 PM](https://user-images.githubusercontent.com/6477701/98894985-5a8b7180-24e9-11eb-9e44-98c5dc2d71ad.png)